### DISCUSSION TITLE
When will the EGSnrc 2022 version be released?

### DISCUSSION THREAD

**Discussion by 1035326373** [nrc-cnrc/EGSnrc#900] [Questions] [answered]
When will the EGSnrc2022 version be released？and what is the difference between EGSnrc2022 and EGSnrc2021？

**Comment by ftessier** [comment]
We are working to integrate changes for the 2022 release of EGSnrc into the `develop` branch. A summary of changes will be provided in the Release notes. In the meantime, you can browse [changes between the `v2021` tag and the `develop` branch](https://github.com/nrc-cnrc/EGSnrc/compare/v2021..develop), and browse the [outstanding pull requests for Release 2022](https://github.com/nrc-cnrc/EGSnrc/pulls?q=is%3Aopen+is%3Apr+milestone%3A%22Release+2022%22).

**Comment by ftessier** [comment, marked as answer]
No EGSnrc version was released in 2022, however [EGSnrc v2023](https://github.com/nrc-cnrc/EGSnrc/releases/tag/v2023) was released on 15 February 2023.
